features/steps/steps.py

```diff
@@ -251,6 +251,12 @@ def when_i_run_command(
             break
 
     full_cmd = prefix + shlex.split(command)
+
+    if "contract_token" in full_cmd:
```

Comment:
I think it is better for us to create a step for the `simulate` command, just to keep this step clean.
We could do a similar approach to the attach step